### ISSUE TITLE
Support for Android and iOS on FS

### ISSUE THREAD

**Issue by paulocoutinhox** [ZhreShold/zupply#13]
Hi,

It has support for file system to Android and iOS?

Thanks.

**Comment by zhreshold** [comment]
No, it does not currently. It is still possible but requires lot's of tests and specialization due to the lack of POSIX compatibility.
